api/base/serializers.py

```diff
@@ -653,7 +676,8 @@ def to_representation(self, obj, envelope='data'):
                         data['embeds'][field.field_name] = result
                 else:
                     try:
-                        data['relationships'][field.field_name] = field.to_representation(attribute)
+                        if not (is_anonymous and hasattr(field, 'view_name') and field.view_name.find('users:') != -1):
```

Comment:
The string matching on a view name terrifies me a little bit. Could you make a base serializer class for all "user serializers" and then do an `not isinstance(field.root, BaseUserSerializer)` Just thinking of the inevitable refactor when someone changes every instance of user to member or something.